onionshare/__init__.py

```diff
@@ -284,10 +371,7 @@ def main(cwd=None):
             print("")
 
     # Start OnionShare http service in new thread
-    t = threading.Thread(
-        target=web.start,
-        args=(app.port, stay_open, common.settings.get("public_mode"), web.password),
-    )
+    t = threading.Thread(target=web.start, args=(app.port,))
```

Comment:
is trailing comma a problem or not a big deal?

Reply:
Actually the trailing comma is required! It's because args must be a tuple, and if I just pass in `(app.port)` it evaluates to `app.port`, which ends up passing in an integer instead. :)